app/assets/stylesheets/site.scss

```diff
@@ -489,3 +489,9 @@ a.form-section-save {
   border: none;
   border-bottom: 1px solid black;
 }
+nav {
```

Comment:
Is this style intended to be scoped for the pagination?
I'm assuming `nav` is used elsewhere.